dbos_transact/context.py

```diff
@@ -19,6 +22,8 @@ def __init__(self) -> None:
 
         self.logger = dbos_logger
 
+        self.request: Optional["Request"] = None
```

Comment:
Do we want to make `Request` the same as tansact-ts? https://docs.dbos.dev/api-reference/contexts#ctxtrequest